src/containers/Transactions/DetailTab.tsx

```diff
@@ -16,7 +16,7 @@ import {
 import './detailTab.scss'
 import { useLanguage } from '../shared/hooks'
 
-const DetailTab = ({ data }) => {
+export const DetailTab: FC<{ data: any }> = ({ data }) => {
```

Comment:
Can we get away from any

Reply:
We can once a better transaction definition exists that we can use.  That will require some extra work.  Right now the priority is getting this to typescript.